CONTRIBUTING.md

```diff
@@ -9,6 +9,8 @@ Frameworks supported by the Open Model Zoo:
 * PyTorch\* (via conversion to ONNX\*)
 * MXNet\*
 
+Open Model Zoo also supports models being converted to ONNX format.
```

Comment:
This is just redundant with what is already written above. ITYM "models already in the ONNX format".

Reply:
Ok, so I won't update CONTRIBUTING.md any how?

Reply:
Well, no, you still need to mention that ONNX models are supported. I suggested you a possible wording.

Reply:
Ok!